.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 node_js:
-  - "8"
+  - "14"
 cache:
   directories:
     - $HOME/.npm
 before_install:
-  - npm install npm@5
+  - npm install npm@7
```

Comment:
Is any of this still necessary?

Reply:
Looks like travisCI will look at `.nvmrc` files but not at package.json `engines` 👉  https://github.com/travis-ci/travis-ci/issues/4653#issuecomment-169035161